app/_data/docs_nav_deck_pre-1.7.yml

```diff
@@ -37,6 +37,8 @@ items:
         url: /guides/best-practices
       - text: Using decK with Kong Gateway (Enterprise)
         url: /guides/kong-enterprise
+      - text: Run decK with Docker
+        url: /guides/run-with-docker
```

Comment:
```suggestion
```
Take it out of this one - there's something up with the pre-1.7 doc set that we need to fix + it's not single-sourced.